Extension/src/LanguageServer/settings.ts

```diff
@@ -712,8 +712,16 @@ export class CppSettings extends Settings {
                         return true;
                     }
                 }
-                if (editorConfigSettings.root?.toLowerCase() === "true") {
-                    return true;
+                if (editorConfigSettings.root !== undefined) {
+                    if (typeof editorConfigSettings.root === "boolean") {
+                        if (editorConfigSettings.root) {
```

Comment:
If the type of this property is boolean, should we just return it instead of adding an `if` here?